Add Mailing list

darkdrgn2k opened the pull request:
Closes #293

Add mailing list to Get Involved
![image](https://user-images.githubusercontent.com/29005789/94754084-38d19300-035e-11eb-85c2-087f58453047.png)

darkdrgn2k committed:
Update get-involved.md

darkdrgn2k committed:
Added mailing list

darkdrgn2k committed:
Update get-involved.md

Shrinks99 commented on get-involved.md:
```diff
@@ -36,3 +36,14 @@ Say hi on our chat {% include icon-matrix.html %}
+  <input style="width:calc(50% - 3px); margin-bottom:8px;" type="email" name="email" value="" placeholder="email@example.com">
+  <input style="width:calc(50% - 3px); margin-bottom:8px;" type="text" name="fullname" value="" placeholder="Your Name (optional)">
+  <input type="hidden" name="digest" value="1">
+  <input style="width:30%;" type="submit" name="email-button" value="Subscribe">
```
I have a few small styling changes I'd like to make before this gets merged if that's cool.  Should get to it next week if we can wait.

Shrinks99 committed:
Mailing Section Fixes

- Moves section to homepage
- Styles according to our existing uI style
- Does not remove in-line style in favour of the actual SCSS files.  Given that we plan on re-doing the website I don't think it's that big a deal though it depends how messy we want to let this website get before then :P

Shrinks99 commented:
Alright, didn't fix the in-line styling but I moved it to the home page and changed the buttons & alignment of some things.

<img width="1230" alt="Screen Shot 2020-10-02 at 11 53 05 PM" src="https://user-images.githubusercontent.com/5672810/94982688-9e628280-050a-11eb-88d8-939ef8b8d1ef.png">
